### PR TITLE
Use native scrollbars in app

### LINKED ISSUE #3993: Accounts - scrollbar gap
When scrollbar is missing in Accounts, there is an empty space, instead of wider account bar.

@matejzak mentioned in #3817
> It would be nice to fix the gap. Although, I suspect we may need to do a bigger refactor to the scrollbars globally.

Screenshot 2021-07-05 at 21 06 04
Screenshot 2021-07-05 at 21 06 00

### PR DESCRIPTION
**Pull request by tomasklim**
closes #3993

- cb5b404 removes custom scrollbar from the app
- b8e484c changes scrollbar of guide panel to be from the top to the bottom

### CODE REVIEW

**Comment by matejkriz** [comment]
snapshots has to be updated, see the failing device-management test: https://gitlab.com/satoshilabs/trezor/trezor-suite/-/jobs/1420442655

File: packages/components/src/components/Scrollbar/index.tsx [hunk 1]
```diff
@@ -1,40 +0,0 @@
...
-
-        /* background of the scrollbar except button or resizer */
-        ::-webkit-scrollbar-track {
-            background-color: transparent;
-        }
-
-        /* scrollbar itself */
-        ::-webkit-scrollbar-thumb {
-            /* 7F7F7F for mac-like color */
-            background-color: ${props => props.theme.SCROLLBAR_THUMB};
```
Comment (matejkriz) [inline packages/components/src/components/Scrollbar/index.tsx:23]: you could also delete `SCROLLBAR_THUMB` color from config

Comment (tomasklim) [reply packages/components/src/components/Scrollbar/index.tsx:23]: 8bcea11f64e81337f5f4c3ad870f4c0f203890b9

**Comment by tomasklim** [comment]
> snapshots has to be updated, see the failing device-management test: https://gitlab.com/satoshilabs/trezor/trezor-suite/-/jobs/1420442655

Snapshot updated. Fixups squashed.

**Review by matejkriz — APPROVED** [review]

**Pull request merged**